### DISCUSSION TITLE
320 Nera doesn't always show charge status

### DISCUSSION THREAD

**Discussion by Routout** [Thomas55555/husqvarna_automower#532] [General]
I have Home Assistant/Node Red automation which sends mower to the dock by return to dock command and switching schedule off when it's raining. When it's dry is turns the schduler again on and sends mower start command if it's within scheduled time.

I think this could be the reason that battery charging status is not charging, altough battery level is increasing while docked.
It have been raining for days, but I can see that in the history there is sometimes Charging status showing.
Is this possible to fix somehow, or is it depending on API?
On Husqvarna app I have 24/7 schedule.

**Comment by Thomas55555** [comment]
It depends on the API. Here is the description: `CHARGING - Mower is currently charging. Note that the mower will only report the activity charging if it was its own decision to go home and charge. I.e., it would like to mow but is charging because it has to. If the mower is restricted it may very well charge when in the charging station, but the activity will be Parked.`

**Comment by Routout** [comment]
Ok. Thanks. 
Maybe if I see it necessary I could make a template sensor or something like that to see charging status. But for now it's not necessary, just wondering behaviour.
